src/fabio/OXDimage.py

```diff
@@ -482,7 +482,7 @@ def dec_TY5(self, stream):
         dim2 = self._shape[0]
         while pos_inp < stream_size and pos_out < array_size:
             if pos_out % dim2 == 0:
-                last = 0
+                last = numpy.int64(0)

```

Comment:
I guess it's only needed there because it's the initialisation
I was first miss-lead by the line above:

```python
        pos_inp = pos_out = current = ex1 = ex2 = 0
```

Reply:
current should also be int64 ... but it is never used for doing math with it.